src/Coordination/CoordinationSettings.cpp

```diff
@@ -134,6 +137,8 @@ void KeeperConfigurationAndSettings::dump(WriteBufferFromOwnString & buf) const
     write_int(coordination_settings->max_requests_batch_size);
     writeText("max_requests_batch_bytes_size=", buf);
     write_int(coordination_settings->max_requests_batch_bytes_size);
+    writeText("max_flush_batch_size=", buf);
```

Comment:
I hope it will not break any of 4LW commands users :D

Reply:
we already expended it multiple times before and didn't have any issues reported for it... for now